src/MSBuild.Conversion.Project/ProjectRootElementExtensionsForConversion.cs

```diff
@@ -110,6 +115,10 @@ public static IProjectRootElement RemoveUnnecessaryPropertiesNotInSDKByDefault(t
                     {
                         // Old MSTest projects specify library, but this is not valid since tests on .NET Core are netcoreapp projects.
                         propGroup.RemoveChild(prop);
+                    }else if (projectStyle == ProjectStyle.WinUI && WinUIFacts.UnnecessaryProperties.Contains(prop.Name, StringComparer.OrdinalIgnoreCase)) 
```

Comment:
style nit, should be on new line

As an aside, you can run the Format Document command in VS over these files as you work and it should come out looking nice.

Reply:
👍